pytest-selenium/tests/conftest.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import pytest
+
+# Window resolutions
+DESKTOP = (1080, 1920)
```

Comment:
Is there a reason these are defined as `height,width` here but they are used in both the code and the docs as `width,height`?

Reply:
Selenium uses these backwards from the way we typically see them. I've adjusted them in the documentation so that they use the typical order.

Reply:
Actually it might be that pytest injects them in the opposite order.